config/app_local.default.php

```diff
@@ -0,0 +1,68 @@
+<?php
```

Comment:
I would prefer this template file to not have .php extension, so that it's clear that it's not actually loaded.

Reply:
Then IDE cant auto hint mistakes when editing it.

Reply:
Why does user need to edit this template file? Isn't this file only used by installer to create `app_local.php`?

Reply:
Contributors here to app do for example.

Reply:
The template file rarely needs update. There's a vastly greater chance of a newbie thinking this file is actually loaded.

Alternative is we can keep the `.php` extension, add it to .gitignore and make the installer delete it after copying.

Reply:
Isnt it a quite common standard with .default files?
I dont see the need of overprotection if we have docs here.
So far the confusion was around other topics, never include of the template file.

Reply:
We can merge this, and have a follow up conversation about the file name ideas there, as those are not related to the issue at hand here IMO.

Reply:
@dereuromark merging into `master` ? or `3.next` ?

Reply:
Since this is skeleton only relevant we can probably do master here just fine.
Doesnt effect existing apps. Only docs would need to be adjusted accordingly.

Reply:
i am using app_local.example.php so far.

Reply:
wordpress uses filename-sample.php
sample or example both work fine. example is a bit longer but has 0 ambiguity where sample has more meanings than example.

Reply:
> Why does user need to edit this template file? Isn't this file only used by installer to create `app_local.php`?

Because you are not foced to use composer create project. You can simply clone and copy the file.